content/en/docs/setup/upgrade/helm/index.md

```diff
@@ -37,6 +37,20 @@ To get started, check out https://istio.io/latest/docs/setup/getting-started/
 
 {{< boilerplate helm-backup >}}
 
+### Upgrading Custom Resource Definitions (CRDs)
+
+{{< warning >}}
+Helm does not [upgrade or delete](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#some-caveats-and-explanations)
+CRDs when performing an upgrade. Given this restriction, additional measures are required to ensure that
+Istio is up to date when performing an upgrade.
+{{< /warning >}}
+
+Istio ships CRDs with its helm charts; they are not templated and are found in the base chart under
+a `crds` folder. However, upgrading Istio with Helm requires a change in workflow different from other
+charts due to the required CRDs be updated for proper functionality.
+
+Please examine the following sections closely as they've been recently been altered.
```

Comment:
Should this also be inside a warning? Maybe move into the warning above. Not sure if that reduces its visibility or not.

Reply:
It would also have to say in which version it was relevant for.

Reply:
There's really no good way to inform users of this change. I'm going to remove this sentence. People that are installing with Helm are hopefully looking back at this document periodically, but there's no way to signal to them that an additional step is necessary when upgrading.